client/modules/datafiles/src/DatafilesToolbar/DatafilesToolbar.tsx

```diff
@@ -34,16 +35,72 @@ export const DatafilesToolbar: React.FC = () => {
       return {
         canPreview:
           selectedFiles.length === 1 && selectedFiles[0].type === 'file',
+        canRename: selectedFiles.length === 1,
         canCopy: user && selectedFiles.length >= 1,
+        canTrash: user && selectedFiles.length >= 1,
       };
     },
     [selectedFiles, user]
   );
 
+  const defaultDestParams = useMemo(
```

Comment:
Do we need to set "default" params for trash here? AFAIK it should always respect the api/system/path from the toolbar props.